app/src/main/java/com/celzero/bravedns/ui/SettingsFragment.kt

```diff
@@ -228,24 +231,18 @@ class SettingsFragment : Fragment(R.layout.activity_settings_screen) {
     private fun handleLockdownModeIfNeeded() {
         val isLockdown = isVpnLockdownEnabled(VpnController.getBraveVpnService())
         if (isLockdown) {
-            b.settingsActivityVpnHeadingText.text = getString(
-                R.string.settings_vpn_heading_disabled)
             b.settingsActivityVpnLockdownDesc.visibility = View.VISIBLE
+            b.settingsActivityExcludeAppsRl.alpha = 0.5f
+            b.settingsActivityAllowBypassRl.alpha = 0.5f
         } else {
-            b.settingsActivityVpnHeadingText.text = getString(R.string.settings_vpn_heading)
             b.settingsActivityVpnLockdownDesc.visibility = View.GONE
+            b.settingsActivityExcludeAppsRl.alpha = 1f
+            b.settingsActivityAllowBypassRl.alpha = 1f
```

Comment:
need to do something similar for on-device blocklists and fav-icon when brave-mode is firewall-only.

Reply:
Handled

Reply:
done